android/KMEA/app/src/main/java/com/keyman/engine/KMManager.java

```diff
@@ -2244,6 +2299,27 @@ public static void applyKeyboardHeight(Context context, int height) {
     }
   }
 
+  public static int getDefaultKeyboardHeight(Context context) {
+    int orientation = getOrientation(context);
+    return getDefaultKeyboardHeight(context, orientation);
+  }
+
+  public static int getDefaultKeyboardHeight(Context context, int orientation) {
+    Resources resources = context.getResources();
+    Configuration originalConfig = resources.getConfiguration();
+    Configuration newConfig = new Configuration(originalConfig);
+    newConfig.orientation = orientation;
+
+    // Create a new context with the updated configuration
+    Context newContext = context.createConfigurationContext(newConfig);
```

Comment:
When the CI runs unit tests, it fails here.
Ever since #5125, there's been problems with unit tests seeing the values defined in the resource files (some mitigation in #8940).

Can get this to run by inserting the following before l. 2308
```java
    if (isTestMode()) {
      // Keyboard height not needed for unit tests  #13578
      return KeyboardHeight_Context_Portrait_Default;
    }
```

Reply:
I can't see your CI, so It's probably better if you add/test this code. I believe you have write access to the branch.

Reply:
You can see the CI build fail with the following:
```bash
./build.sh build:engine build:app test:engine --debug
```

Reply:
Done.